### ISSUE TITLE
How to train LinearProjectionDistogramModel for my data？

### ISSUE THREAD

**Issue by hasaiki136** [facebookresearch/esm#655]
Hello, thanks for your great work.

In your lm_design folder, there are two tasks - fixedbb and free_generation. For the first task, I want to change the weight for LinearProjectionDistogramModel to predict my proteins distance bins accurately. I tried to train the model on my protein data, with learning rate of 1e-2 using cce_loss (refer to your loss.py) between all [i, j] pairs in the predicted distogram and ground truth distogram. But didn’t get expected results.

So I hope to know the way you train the LinearProjectionDistogramModel, how you calculate the total loss and how to define the accuracy ?
